### PR TITLE
update gitbook version

### PR DESCRIPTION
**Pull request by hellonico**
to version 2.0.1

(version 2.0.2 is broken)

### CODE REVIEW

**Comment by tobegit3hub** [comment]
Thanks for your contribution. I knew that version 2.x.x had known problem to setup gitbook. I will test 2.0.1 and merge this later :smile:

**Comment by tobegit3hub** [comment]
I think 2.x has the new library to install gitbook client. And I got this error. It would be better to fix it.

```
➜  gitbook-server git:(master) docker logs 90c539fadc1d6c67f418019c63e07614788870c080a926155b9208469fb1aefd
You need to install 'gitbook-cli' to have access to the gitbook command anywhere on your system.
If you've installed this package globally, you need to uninstall it.
>> Run 'npm uninstall -g gitbook' then 'npm install -g gitbook-cli'
```

**Comment by hellonico** [comment]
Hi, 
Sorry for the late reply.
I just pushed the proper install of gitbook-cli/gitbook version 2.0.2

It works for me, but ping me if you have problems with the latest commit.

File: Dockerfile [hunk 1]
```diff
@@ -3,14 +3,14 @@ MAINTAINER tobe tobeg3oogle@gmail.com
...
+# Install dependencies
+RUN apt-get install -y curl git calibre && \
+	curl -sL https://deb.nodesource.com/setup | bash - && \
+	apt-get install -y nodejs && \
+	npm install -g gitbook-cli
 
-# Support gitbook pdf
-RUN apt-get install -y calibre
+# Install latest version
+RUN gitbook install 2.0.2
```
Comment (tobegit3hub) [inline Dockerfile]: I'm a little confused about this. Anyway, I will test later and really thanks for your contribution :smile:

**Comment by hellonico** [comment]
I am installing calibre at the very beginning, and one needs to install gitbook through the gitbook-cli command. ([how to use it](https://github.com/GitbookIO/gitbook#how-to-use-it)

**Comment by tobegit3hub** [comment]
It works like a charm. Thanks!

I'm gonna add `npm install svgexport -g` so that everyone can use `gitbook pdf` easily.

**Pull request merged**